logging/bin/deploy_opensearch.sh

```diff
@@ -73,13 +73,13 @@ if [ ! -f  $TMP_DIR/es-transport.pem ]; then
    log_debug "Extracting es-transport cert from secret"
    kubectl -n $LOG_NS get secret es-transport-tls-secret -o=jsonpath="{.data.tls\.crt}" |base64 --decode > $TMP_DIR/es-transport.pem
 fi
-node_dn=$(openssl x509 -subject -nameopt RFC2253 -noout -in $TMP_DIR/es-transport.pem | sed  "s/subject=\s*\(\S*\)/\1/")
+node_dn=$(openssl x509 -subject -nameopt RFC2253 -noout -in $TMP_DIR/es-transport.pem | sed  "s/subject=\s*\(\S*\)/\1/" | sed -e 's/^[ \t]*//')
 
```

Comment:
What is the "-e" option on sed doing?

Reply:
Was trying to use "-e" to chain sed commands. WIll clean this up

Reply:
"-e" will let you chain sed expressions, opted for that rather than piping in this case